tracer/test/test-applications/integrations/Samples.VersionConflict.2x/Samples.VersionConflict.2x.csproj

```diff
@@ -2,7 +2,7 @@
 
   
   <ItemGroup>
-    <PackageReference Include="Datadog.Trace" Version="2.255.249" />
+    <PackageReference Include="Datadog.Trace" Version="2.255.248" />
```

Comment:
The package bump was made so that the version used by the application would have the updated definition for `IDistributedTracer.SetSpanContext(IReadOnlyDictionary<string, string> value)` so it can duck cast the automatic instrumentation's `AutomaticTracer`

Reply:
I think updating the nuget package was required to fix the behavior, but not for the ducktyping 😃  Only `IAutomaticTracer`/`ICommonTracer` are ducktyped, changing `IDistributedTracer` is not a breaking change across versions of the tracer.